Code/GraphMol/MolStandardize/AcidBaseCatalog/acid_base_pairs.in

```diff
@@ -0,0 +1,54 @@
+//
+//  Copyright (C) 2021 Greg Landrum
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+// The default list of AcidBasePairs, sorted from strongest to weakest.
+// This list is derived from the Food and Drug: Administration Substance
+// Registration System Standard Operating Procedure guide.
+//
+//	Name	Acid	Base
+std::vector<std::tuple<std::string, std::string, std::string>>
```

Comment:
these should be probably be in an unnamed namespace const and possibly static.

I think you can wrap the #include in an unnamed namespace, this may help with the java failure.

Reply:
Good point about const and the namespace.
The problems with the linux java build looks like a compiler thing, but let's see if making it `const` helps the compiler

Reply:
Ok, the problem with the java build here is almost certainly caused by the compiler (gcc 5.5 on that system).

Reply:
You could just make it a struct of c-strings so that it lives in the text section and doesn't have to make the pesky strings.

Reply:
and, confirmed that this works fine with the same compiler if I use `-std=c++17`
grrrrr
I will take a look at how ugly using `const char *`s will make the code

Reply:
nope. that won't work without a bunch of ugly work or memory leakage. 
The JSON parser needs to be able to set `CleanupParameters::acidbaseData` and there's no straightforward way to handle the memory ownership if we're using `const char *` there.
I'll explore a bit more